src/plugins/data/public/ui/query_string_input/_query_bar.scss

```diff
@@ -61,7 +72,6 @@
     // sass-lint:disable-block no-important
     flex-grow: 0 !important;
     flex-basis: auto !important;
-    margin-right: -$euiSizeXS !important;
```

Comment:
Is this due to changes in the components?

Reply:
The theme used to have
```
.euiSuperDatePicker__flexWrapper--noUpdateButton {
  ...
  max-width: 100%;
}
```
which necessitated a negative `margin-right` to reduce the gap between the data selection and the "Refresh" button.

The newer EUI has removed the `max-width: 100%` to allow for 
```
.euiSuperDatePicker__flexWrapper {
  max-width: calc(100% + #{$euiSizeS});
  ...
}
```
to be applied here which eliminates the need for the negative margin.